### ISSUE TITLE
[FR] Disable or hide or not to render error containers

### ISSUE THREAD

**Issue by muphblu** [guillaumepotier/Parsley.js#1187]
What kind of issue is this? (put 'x' between the square brackets)
 - [x] Feature Request. Make sure there's no good way to do what you want first;
       consider asking on http://stackoverflow.com/questions/ask first.

It is immpossible to disable error container rendering, except off setting `uiEnabled: false`.
But `uiEnabled: false`, also disables events on ui(`triggerAfterFailure` option). 

So in my situation when i want to use parsley to validate but not to show validation results, i didn't find nothing better than set `uiEnabled: false`, but to hide `errorsWrapper` through css. It works, but looks more like a hack than a solution.

Summary:
- error container renders always if `uiEnabled: true`;
- `uiEnabled: false` and `triggerAfterFailure: 'input change'` not work simultaneously
```
form.parsley({
                triggerAfterFailure: 'input change',
                uiEnabled: true,
                errorsWrapper: '<ul class="parsley-errors-list hidden help-block"></ul>'
            })
```

P.S. I don't know may be `uiEnabled: false` shoudn't work this way and it's a bug.

**Comment by marcandre** [comment]
Thanks for the request.

First, `uiEnabled` plays no logical role I can think of, you are right. I should probably deprecate it.

IMO, hidding the generated errors with CSS doesn't seem like a terrible hack.

If not outputting error messages was a trivial feature to add I might, but it isn't right now (Parsley uses the list of errors to know the current state).
I'd be more interested in knowing how you are displaying errors. Maybe there's an alternate way that could be supported directly? Please check #922

**Comment by muphblu** [comment]
Thank you for the reply)

We use only simple validations like required and datetime, so for us in most cases there is no need to display errors in the way parsley do it. We decided simply higlight fields and labels by red color using [bootstrap](http://getbootstrap.com/css/#forms-control-validation).

I think FR could be closed, for me it very similar to #922
